src/proto/Makefile

```diff
@@ -43,14 +43,16 @@ GO_CONTROL_FILES = common/proto/mgmt/acl.pb.go\
 		   common/proto/mgmt/pool.pb.go\
 		   common/proto/mgmt/srv.pb.go\
 		   common/proto/mgmt/storage_query.pb.go\
+		   common/proto/mgmt/rank.pb.go\
 		   common/proto/ctl/storage.pb.go\
 		   common/proto/ctl/common.pb.go\
 		   common/proto/ctl/storage_nvme.pb.go\
 		   common/proto/ctl/storage_scm.pb.go\
 		   common/proto/ctl/system.pb.go\
 		   common/proto/ctl/control.pb.go\
- 		   common/proto/ctl/network_scan.pb.go\
+		   common/proto/ctl/network_scan.pb.go\
 		   common/proto/srv/srv.pb.go\
+		   common/proto/ctl/rank.pb.go\
```

Comment:
Hm, when I asked for this I assumed that it was trivial to import protobuf files from other directory locations. If we wind up duplicating either way, maybe it should just be left the way it already was.